pages/news/2020-08-XX-new-in-2-3-deck-clone.html

```diff
@@ -0,0 +1,53 @@
+title: New In 2.3: Deck Cloning
+author: Matthew Nicholson
+date: 2020-08-XX 01:00:00
```

Comment:
Please fill in the date and rename this file accordingly, then we can merge.

Reply:
is the missing date preventing the netlify preview of this post?
https://deploy-preview-136--mixxx-website.netlify.app/news/

Reply:
yep, it is. at least with a proper date also the local `cactus` preview works.

Reply:
If someone reviewed and merged https://github.com/mixxxdj/website/pull/148 this wouldn't be an issue anymore.